### ISSUE TITLE
About the loss Lc

### ISSUE THREAD

**Issue by wbwang17** [liuxiaoyu1104/SmartControl#7]
Thanks for the great job. But in the code, I cannot find the Lc loss (Eq. 5). Could you help me locate it?

**Comment by liuxiaoyu1104** [comment]
Hello! In the future, I will update my training code to add the Lc Loss part. Additionally, using Loss_sd can achieve good results; the gains from the L_c part are not as significant, so you can train with this code first.
